File: CPE-1star/UVA10268-498' .py
#按照他給的第二個式子求導數即可
while True:
	try:
		x=int(input())
		nums=list(map(int,input().split()))
		n=len(nums)-1
		subn=n-1
		ans=0
		for i in range(len(nums)):
			ans+=n*(nums[i]*(x**subn))
			n-=1
			subn-=1
		print(int(ans))
	except EOFError:
		break

#2
while True:
	try:
		x=int(input())
		n=list(map(int,input().split()))
		nn=len(n)-1
		nnn=len(n)-2
		ans=0
		for i in range(len(n)-1):
			ans+=nn*n[i]*(x**nnn)
			nn-=1
			nnn-=1
		print(int(ans))
	except:
		break

File: CPE-1star/UVA10268-498'.cpp
#include<iostream>
#include<vector>
#include<cmath>
using namespace std;
int main(){
    int x;
    while(cin>>x){
        vector<int> nums;
        int num;
        while(cin>>num){
            nums.push_back(num);
            if(getchar()=='\n')
                break;
        }
        int n=nums.size()-1,subn=nums.size()-2;
        int ans=0;
        for(int i=0;i<nums.size();i++){
            ans+=n*(nums[i]*(int)pow(x,subn));
            n-=1;
            subn-=1;
        }
        cout<<ans<<"\n";
    }
    return 0;
}